tutorials/frontend/deploy_ssd_gluoncv.py

```diff
@@ -44,8 +47,8 @@
     'ssd_512_resnet50_v1_voc',
     'ssd_512_resnet50_v1_coco',
     'ssd_512_resnet101_v2_voc',
-    'ssd_512_mobilenet1_0_voc',
-    'ssd_512_mobilenet1_0_coco',
+    'ssd_512_mobilenet1.0_voc',
+    'ssd_512_mobilenet1.0_coco',
 ]
 
 model_name = "ssd_512_resnet50_v1_voc"
```

Comment:
Can we use
``` python
model_name = supported_model[2]
```
, and remove `ssd_512_resnet18_v1_voc` and `ssd_512_resnet18_v1_coco` in `supported_model` since they are not available in current `model_store` in gluoncv? [See #2822]

Reply:
Sure, I'll do the modifications in the PR according to yours.